configs/sklearn_hist_boosting.json

```diff
@@ -0,0 +1,305 @@
+                        "x": "data/higgs1m_x_train.npy",
+                        "y": "data/higgs1m_y_train.npy"
+                    },
+                    "testing": {
+                        "x": "data/higgs1m_x_test.npy",
+                        "y": "data/higgs1m_y_test.npy"
+                    }
+                }
+            ],
+            "max-bin": 255,
+            "learning-rate": 0.1,
+            "reg-lambda": 1,
+            "max-depth": 8,
+            "max-leaves": 256,
+            "n-estimators": 1000,
+            "objective": "binary:logistic"
+        },
+        {
+            "dataset": [
+                {
+                    "source": "npy",
+                    "name": "letters",
+                    "training": {
+                        "x": "data/letters_x_train.npy",
+                        "y": "data/letters_y_train.npy"
+                    },
+                    "testing": {
+                        "x": "data/letters_x_test.npy",
+                        "y": "data/letters_y_test.npy"
+                    }
+                }
+            ],
+            "learning-rate": 0.03,
+            "max-depth": 6,
+            "n-estimators": 1000,
+            "objective": "multi:softprob"
+        },
+        {
+            "dataset": [
+                {
+                    "source": "npy",
+                    "name": "mlsr",
+                    "training": {
+                        "x": "data/mlsr_x_train.npy",
+                        "y": "data/mlsr_y_train.npy"
+                    }
+                }
+            ],
+            "max-bin": 255,
+            "learning-rate": 0.3,
+            "reg-lambda": 2,
+            "max-depth": 8,
+            "n-estimators": 200,
+            "objective": "multi:softprob"
+        },
+        {
+            "dataset": [
+                {
+                    "source": "npy",
+                    "name": "mortgage1Q",
+                    "training": {
+                        "x": "data/mortgage1Q_x_train.npy",
+                        "y": "data/mortgage1Q_y_train.npy"
+                    }
+                }
+            ],
+            "n-estimators": 100,
+            "objective": "reg:squarederror",
+            "max-depth": 8,
+            "learning-rate": 0.1,
+            "reg-lambda": 1,
+            "max-leaves": 256
+        },
+        {
+            "dataset": [
+                {
+                    "source": "npy",
+                    "name": "plasticc",
+                    "training": {
+                        "x": "data/plasticc_x_train.npy",
+                        "y": "data/plasticc_y_train.npy"
+                    },
+                    "testing": {
+                        "x": "data/plasticc_x_test.npy",
+                        "y": "data/plasticc_y_test.npy"
+                    }
+                }
+            ],
+            "n-estimators": 60,
+            "objective": "multi:softprob",
+            "max-depth": 7
```

Comment:
What about "subsample", "colsample-bytree"?
Are they missing in sklearn gb?

Reply:
This applies to all cases

Reply:
These parameters are not supported by sklearn.